core/src/main/java/org/apache/carbondata/core/metadata/schema/table/DiskBasedDMSchemaStorageProvider.java

```diff
@@ -175,6 +175,18 @@ public DiskBasedDMSchemaStorageProvider(String storePath) {
     }
   }
 
+  private void checkAndReloadDataMapSchemasWithoutNewFile() throws IOException {
```

Comment:
It is duplicated to below method, just pass boolean to handle in below method instead of duplicating it